#1 Update README

sijakob opened the pull request:
Added the WHAT, WHY and HOW.

sijakob committed:
#1 Update README

Added the WHAT, WHY and HOW.

WhiteMaken reviewed (changes requested):
Why part is too vague in terms of specifications
Multiple typos

WhiteMaken committed:
#1 Modify the why section of the README

Changed the why? section of the README to be more specific about which kind of technology we plan on using during the project.
